### PR TITLE
bands: tested with cp2k 7.1 and 8.2, fixed possible issue with 8.1

### LINKED ISSUE #65: Fix example_bands test
For some reason the [`example_bands`](https://github.com/aiidateam/aiida-cp2k/blob/a5c0d7064a42e54e45985c77adde6e53bc0cf6ae/examples/single_calculations/fixme_bands.py) test is failing, so I renamed it to `fixme_bands`.

- [ ] Understand whether the problem is caused by CP2K itself or by the wrong input prepared by plugin.

- [ ] Rename the test back to `example_bands.py`

### PR DESCRIPTION
**Pull request by dev-zero**
CP2K ~8.1 does not print weights for each KP, hence the selection of values could be wrong, make it more robust. Fixes #65

### CODE REVIEW

**Review by yakutovicha — COMMENTED** [review]
@dev-zero, sorry for the long wait. For an unknown reason, the cp2k fails in the docker container to compute the bands. I will address that in a separate PR.

Just rename `example_bands` back to `fixme_bands` and I will approve/merge.

**Comment by dev-zero** [comment]
found the issue, fixed it, see comments

**Comment by dev-zero** [comment]
TL;DR the version provided with Ubuntu terminates with a Floating Point Exception, not much to be done about it.

**Comment by yakutovicha** [comment]
> TL;DR the version provided with Ubuntu terminates with a Floating Point Exception, not much to be done about it.

Yeah, that is why I proposed earlier to not worry about it. I think we can solve the issue by installing cp2k from [conda-forge](https://anaconda.org/conda-forge/cp2k).

I can do it in the next PR.

**Review by yakutovicha — APPROVED** [review]
